spot-setup/README.md

```diff
@@ -28,6 +32,11 @@ This file also contains sources desired to be installed as part of Apache Spot,
 
 To read more about these variables, please review the [wiki] (https://github.com/Open-Network-Insight/open-network-insight/wiki/Edit%20Solution%20Configuration).
 
+## Install
+
+1. run `sudo ./install.sh`
+2. run `hdfs_setup.sh`
```

Comment:
In case users want to land in different nodes, then they would need to run separate scripts but if they do in a single node, then just run install-all.sh, is that what you are saying?